source/beamline/PGM/PGMBeamline.cpp

```diff
@@ -147,6 +169,18 @@ void PGMBeamline::setupExposedDetectors()
 	addExposedDetector(i0EndstationBladeCurrentDetector_);
 	addExposedDetector(i0BeamlineBladeCurrentDetector_);
 	addExposedDetector(photodiodeBladeCurrentDetector_);
+
+	addExposedDetector(oceanOpticsDetector_);
+}
+
+bool PGMBeamline::getConnectedState() const
```

Comment:
never mind... there is a isConnected() function, which returns the `connected_` flag. 

We can solve this when we have confliction...